src/encoded/static/components/filegallery.js

```diff
@@ -1985,7 +1992,17 @@ const TabPanelFacets = (props) => {
     // We do not count how many results there are for a given assembly because we will not display the bars
     fileList.forEach((file) => {
         if (file.genome_annotation && file.assembly && !assembly[`${file.assembly} ${file.genome_annotation}`]) {
-            assembly[`${file.assembly} ${file.genome_annotation}`] = +file.assembly.match(/[0-9]+/g);
+            const assemblyNumber = file.assembly.match(/[0-9]+/g);
+            const annotationNumber = +file.genome_annotation.match(/[0-9]+/g);
+            let annotationDecimal = 0;
```

Comment:
This was the other thing that I thought perhaps should be a function. I don't know how I would know which of the elements in the .match() result would be the right one, so I'm not sure if my update will necessarily improve the situation if there's a new annotation with two numerical components, but there would be a chance it would be correct! Possibly I should print a warning here also? A check for a match with more than one result? I also added the list of assembly / annotations from Idan so we have a record of that that is correct for now.